src/explore-education-statistics-admin/src/pages/release/ReleaseStatusPage.tsx

```diff
@@ -169,6 +170,8 @@ const ReleaseStatusPage = () => {
                       </td>
                       <td>{status.approvalStatus}</td>
                       <td>{status.internalReleaseNote}</td>
+                      <td>{`${status.releaseVersion + 1}`}</td>{' '}
+                      {/* +1 because version starts from 0 in DB */}
```

Comment:
Just wondering if there's anything in `ReleaseStatusPage.test.tsx` or UI tests that can be updated for this?